dbfit-java/build.gradle

```diff
@@ -83,7 +83,10 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task dbConnectionProperties {
+    }
+
+    task integrationTest(type: Test, dependsOn: dbConnectionProperties) {
```

Comment:
IntegrationTest depends on dbConnectionProperties which includes some admin user/pass... I'm not sure I'm following the whole chain correctly but do we really need admin account to run the integration tests?